portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,38 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomFact() {
+  const facts =
+      ['My favorite food is pasta.', 
+      'One place I really want to visit is Brazil.', 
+      'I know a little bit of french from my highschool language classes.', 
+      'One CS project I am very proud of is coding a sensor for my robotics team.'];
+
+  // Pick a random greeting.
+  const fact = facts[Math.floor(Math.random() * facts.length)];
+
+  // Add it to the page.
+  const factContainer = document.getElementById('greeting-container');
+  factContainer.innerText = fact;
+}
+
+function addRandomPlantPicture() {
+    const plant_pics = [
+        "/images/seedling.jpg", "/images/garfield.jpeg",
+        "/images/flowers.jpeg"
+    ];
+
+    const picture = plant_pics[Math.floor(Math.random() * plant_pics.length)];
+    const picContainer = document.getElementById('plant_pic');
+    document.getElementById('plant').src = picture;
+
+}
+
+async function serverHelloWorld() {
```

Comment:
nit: "serverHelloWorld" doesn't really explain what's going on in the function that well. NBD for now but naming it something like displayHelloWorld or showHelloWorld make it a little easier to understand when reading the html that calls this function :)

Reply:
Changed this in Jennifer_branch_week 2